### ISSUE TITLE
Package description should be updated to refer to this project.

### ISSUE THREAD

**Issue by dmitshur** [gopheracademy/congo#21]
See http://godoc.org/github.com/gopheracademy/congo.

The package comment in [`doc.go`](https://github.com/gopheracademy/congo/blob/d6b0f318a79c45ecd9d1c627cfb5854f91ec89fa/doc.go#L3-L9) should be updated to refer to congo and describe it. :)

This is trivial to fix, and I'd make a PR, I just don't know how exactly you'd prefer to word the description.

**Comment by dmitshur** [comment]
@bketelsen, would a PR be helpful? The only reason I didn't make one is because I don't know the best wording you'd want to use, but I can take something from the README, Wiki, or repo description. For example:

``` Go
// Congo is an open source conference management platform.
package main
```

How's that?

**Comment by dmitshur** [comment]
Fixed in 296cf208e0cbf0fb2266df9e49db2cb70e60d4b1.
